evaluation/swe_bench/README.md

```diff
@@ -68,20 +68,41 @@ temperature = 0.0
 
 ## Test if your environment works
 
+Make sure your Docker daemon is running, and you have pulled the `eval-swe-bench:full-v1.0`
+docker image. Then run this python script:
+
 ```bash
-python3 evaluation/swe_bench/swe_env_box.py
+poetry run python evaluation/swe_bench/swe_env_box.py
 ```
 
-If you get to the interactive shell successfully, it means success!
+If you get to the interactive shell successfully, it means your environment works!
+If you see an error, please make sure your `config.toml` contains all
+`SWEBench eval specific` settings as shown in the previous section.
 
 ## Run Inference on SWE-Bench Instances
 
 ```bash
-./evaluation/swe_bench/scripts/run_infer.sh eval_gpt4_1106_preview
+./evaluation/swe_bench/scripts/run_infer.sh [model_config] [agent] [eval_limit]
```

Comment:
Maybe we could provide an example command (commented out under this line)? like

`./evaluation/swe_bench/scripts/run_infer.sh eval_gpt4_1106_preview CodeActAgent 300` as an example?

Reply:
See L104 if you missed it :)

Reply:
Yep! Just next line maybe :D -- but it shouldn't matter too much

Reply:
make sense, added.

Reply:
LGTM! Let's merge!